.github/workflows/docs.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: '10.x'
+          node-version: '12.x'
```

Comment:
Should we move directly to 14 instead? It is the current active LTS release. Would it affect us?

Reply:
From @monitz87: "Node 14 resulted in warnings when using commitizen, so I decided to go with 12 just to be safe"